indigo-frontend/src/core/types/entities/template.i.ts

```diff
@@ -1,63 +1,29 @@
+import { ExperimentStatus } from '@/core/enums/experiment-status.enum';
 import { BaseEntity } from './base-entity.i';
+import { Attachment } from './attachment.i';
+import { ProjectAcl } from './acl.i';
 
-export interface TemplateComponentAttachments {
-  type: 'attachments';
-}
-export interface TemplateComponentBatches {
-  type: 'batches';
-}
-export interface TemplateComponentConceptDetails {
-  type: 'conceptDetails';
-}
-export interface TemplateComponentExperimentDetails {
-  type: 'experimentDetails';
-}
-export interface TemplateComponentExperimentDescription {
-  type: 'experimentDescription';
-}
-export interface TemplateComponentPreferredCompoundsDetails {
-  type: 'preferredCompoundsDetails';
-}
-export interface TemplateComponentPreferredCompoundsSummary {
-  type: 'preferredCompoundsSummary';
-}
-export interface TemplateComponentReactionDetails {
-  type: 'reactionsDetails';
-}
-export interface TemplateComponentStoichiometryTable {
-  type: 'stoichiometryTable';
-  reactantsReagentsSolvents: boolean;
-  reactionProducts: boolean;
-}
-export interface TemplateComponentReactionScheme {
-  type: 'reactionScheme';
-}
-export interface TemplateComponentReactants {
-  type: 'reactants';
-}
-export interface TemplateComponentIntendedProducts {
-  type: 'intendedProducts';
-}
-export type TemplateComponent =
-  | TemplateComponentAttachments
-  | TemplateComponentBatches
-  | TemplateComponentConceptDetails
-  | TemplateComponentExperimentDetails
-  | TemplateComponentExperimentDescription
-  | TemplateComponentPreferredCompoundsDetails
-  | TemplateComponentPreferredCompoundsSummary
-  | TemplateComponentReactionDetails
-  | TemplateComponentStoichiometryTable
-  | TemplateComponentReactionScheme
-  | TemplateComponentReactants
-  | TemplateComponentIntendedProducts;
-
-export interface TemplateTab {
+export interface TeamMember {
   name: string;
-  components: TemplateComponent[];
+  email: string;
+  role: string;
+  avatar: string;
+  userId?: string;
+  inherited?: boolean;
 }
 
 export interface Template extends BaseEntity {
```

Comment:
Seems not correct. Please refer to https://indigo-eln-dev.test.lifescience.opensource.epam.com/swagger/eln/#/Template%20Resource/get_api_eln_templates